recipes-extended/onl/onl_git.bb

```diff
@@ -62,3 +80,7 @@ ONL_PLATFORM_SUPPORT:x86-64 = " \
     x86-64-delta-ag5648-r0 \
     x86-64-delta-ag7648-r0 \
 "
+
+ONL_MODULE_VENDORS:x86-64 = " \
```

Comment:
An empty ONL_MODULE_VENDORS should be created in the commit "onl: add support for building common vendor modules" already. Otherwise, the commit does not show how and where the newly introduced variable is meant to be used, and when it does get used in "agema-ag5648: build platform modules via ONL", there is no indication where the variable added to onl_git.bb suddenly came from.

Also, there should be a brief comment above `ONL_MODULE_VENDORS:x86-64 =` indicating what the variable is good for, because from the variable name it is not obvious at all.